packages/core/src/renderers/systems/textures/TextureSystem.js

```diff
@@ -1,7 +1,34 @@
 import WebGLSystem from '../WebGLSystem';
-import GLTexture from './GLTexture';
 import { removeItems } from '@pixi/utils';
 
+class GLTexture
```

Comment:
Preference would be to externalize GLTexture to it's only file since it's not just an internal class and is shared. Also, need at least a class definition here. What about a `destroy` method which would take care of `texture._glTextures[this.renderer.CONTEXT_UID].texture.destroy();` below, but you could also nullify the `texture` property.